R/xyPlot.R

```diff
@@ -0,0 +1,29 @@
+#' xyPlot
+#'
+#' returns scatter plot for user-defined params and subset criteria
+#' 
+#' @param df data 
+#' @param x_arg chr, the x-axis parameter to be plotted 
+#' @param y_arg chr, the y-axis parameter to be plotted 
+#' @param z_subset chr, subsetting criteria
+#' 
+#' 
```

Comment:
`year_outlier_plot`...that is...we need a more descriptive function name

Arguments:
`area`, with an NA option that will bring all areas, or character vector to subset the data

`year.x.y` = c(2010, 2005) strict 2 year vector that defines what data years to use (and defines which year is the x side and which is y)

Include 1:1 line, and force scales to be the same in x and y (so...no user arguments here).